rmgpy/data/kinetics/family.py

```diff
@@ -1521,6 +1521,19 @@ def apply_recipe(self, reactant_structures, forward=True, unique=True):
                 atom_labels['*2'].label = '*3'
                 atom_labels['*3'].label = '*2'
 
+            elif label == 'surface_abstraction':
+                atom_labels['*1'].label = '*3'
+                atom_labels['*2'].label = '*5'
+                atom_labels['*3'].label = '*1'
+                atom_labels['*5'].label = '*2'
+
+            elif label == 'surface_abstraction_single_vdw':
```

Comment:
in #2081 I came up with a different set of label swaps here
https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2081/files
Please can you double check which of us is wrong? (or are they in fact equivalent?)

Reply:
yours is the right way.  Do you want to merge that PR in first, or maybe cherrypick that commit onto this one?

Reply:
just cherrypicked that commit and rebased